community/postsrsd/postsrsd.initd

```diff
@@ -0,0 +1,29 @@
+#!/sbin/openrc-run
+
+description="Postfix SRS Daemon"
+pidfile="/run/postsrsd.pid"
+command="/usr/sbin/postsrsd"
+command_args="-D -d ${SRS_DOMAIN} -s ${SRS_SECRET} -f ${SRS_FORWARD_PORT}
+              -r ${SRS_REVERSE_PORT} -p \"${pidfile}\" -u ${RUN_AS}
+              -c \"${CHROOT}\" -a \"${SRS_SEPARATOR}\""
```

Comment:
@jirutka @clandmeter I've updated the postsrsd confd file to include defaults now. The only thing we cannot provide defaults for is `SRS_DOMAIN`, users have to provide that their selves.

The capitalization of the variables comes from upstream, so I'm just referencing them.